client/my-sites/current-site/notice.jsx

```diff
@@ -52,6 +52,28 @@ export class SiteNotice extends Component {
 		);
 	}
 
+	getLaunchpadNotice() {
+		const { site } = this.props;
+
+		return (
+			<UpsellNudge
+				callToAction="Next Steps"
```

Comment:
Do we want to add translation on line 60 and line 72

Reply:
Ah good catch. Took out translations temporarily but we def want to add them back

Reply:
I'll fix this when I'm back from lunch